http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/sam/SamAuthProvider.scala

```diff
@@ -190,33 +174,10 @@ class SamAuthProvider(val config: Config, serviceAccountProvider: ServiceAccount
     * @return A Future that will complete when the auth provider has finished doing its business.
     */
   override def notifyClusterDeleted(internalId: ClusterInternalId, userEmail: WorkbenchEmail, creatorEmail: WorkbenchEmail, googleProject: GoogleProject, clusterName: ClusterName)
-                                   (implicit executionContext: ExecutionContext): Future[Unit] = {
-    retryUntilSuccessOrTimeout(shouldInvalidateSamCacheAndRetry, s"SamAuthProvider.notifyClusterDeleted call failed for ${googleProject.value}/${clusterName.value}")(samRetryInterval, samRetryTimeout) { () =>
-      Future {
-        blocking(samClient.deleteNotebookClusterResource(internalId, creatorEmail, googleProject, clusterName))
-      }.recover {
-        // treat 404s from Sam as the cluster already being deleted
-        case e: ApiException if e.getCode == StatusCodes.NotFound.intValue => ()
-        case e if shouldInvalidateSamCacheAndRetry(e) =>
-          // invalidate the pet token cache between retries in case it contains stale entries
-          // See https://github.com/DataBiosphere/leonardo/issues/290
-          samClient.invalidatePetAccessToken(creatorEmail, googleProject)
-          throw e
-      }
-    }
-  }
-
-  private def shouldInvalidateSamCacheAndRetry(t: Throwable): Boolean = {
```

Comment:
We decided to remove this since we weren't sure if it was still needed right? I'm 👍 on this.